utils/node_local_test.py

```diff
@@ -6350,25 +6496,50 @@ def run(wf, args):
                         fatal_errors=fatal_errors) as server:
             if args.mode == 'launch':
                 run_in_fg(server, conf, args)
-            elif args.mode == 'overlay':
+            elif args.mode == 'overlay' and 'special_dfuse_overlay' in special_list:
                 fatal_errors.add_result(run_duns_overlay_test(server, conf))
             elif args.mode == 'set-fi':
                 fatal_errors.add_result(server.set_fi())
             elif args.mode == 'all':
                 fi_test_dfuse = True
-                fatal_errors.add_result(run_posix_tests(server, conf))
-                fatal_errors.add_result(run_dfuse(server, conf))
-                fatal_errors.add_result(run_duns_overlay_test(server, conf))
+                fatal_errors.add_result(run_posix_tests(server, conf, test_dict.keys()))
+                if 'special_dfuse_multi' in special_list:
+                    fatal_errors.add_result(run_dfuse(server, conf))
+                if 'special_dfuse_overlay' in special_list:
+                    fatal_errors.add_result(run_duns_overlay_test(server, conf))
```

Comment:
I think of these two as legacy tests and they just want re-writing as methods on `posix_tests`